__tests__/__mocks__/fileMock.ts

```diff
@@ -1,10 +1,12 @@
 // __mocks__/fileMock.ts

```

Comment:
It's just a reordering to have constants in alphabetical order plus added to constants: `voiceIcon` and `voiceIconDisabled`